src/ErrorPage/styled.js

```diff
@@ -0,0 +1,37 @@
+import styled from "styled-components";
+
+export const Container = styled.div`
+  padding: 180px 16px 32px;
+  text-align: center;
+
+  @media (max-width: ${({ theme }) => theme.media.mobile}px) {
+    margin: auto;
+  }
```

Comment:
I think that in this case the margin would be better replaced by padding: 16px 16px 32px;
This gives a nicer effect on mobile devices.

Reply:
In my opinion, it looks better, when it's vertically centered:)
Layout at the top:
![layout at the top](https://github.com/wroblewski82/movies-browser/assets/124407697/5495fc3f-8328-4e5c-9ad0-b71c5804bd4f)
and vertically centred:
![layout vertically centred](https://github.com/wroblewski82/movies-browser/assets/124407697/05e176b9-42d9-44da-b052-f755b3faacf0)